### PR TITLE
aspects: Update team information

### PR DESCRIPTION
**Pull request by techievena**
Add information about the ongoing projects and team members of the aspects team

### CODE REVIEW

**Comment by techievena** [comment]
@coala/aspects-developers Please review

File: README.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+Github profile page: https://github.com/userzimmermann/
+Gitlab profile page: https://gitlab.com/userzimmermann/
+
+The mother of all futures of coala! [#aspectsYEAH](https://github.com/orgs/coala/projects/3)
+
+The aspects team is responsible for implementing aspects in coala as defined in
+[cEP-0005](https://github.com/coala/cEPs/blob/master/cEP-0005.md). The ongoing
+projects of the aspects team are:
+-  https://github.com/coala/coala/projects/2
+-  https://github.com/coala/coala-bears/projects/1
```
Comment (userzimmermann) [inline README.md]: I want to remove the above 2 repo based projects in favor of the org wide `#aspectsYEAH` project below. So better remove those links

**Review by userzimmermann — CHANGES_REQUESTED** [review]
@Techievena Thanks a lot for those additions! Just remove the links to the obsolete repo based `#aspectsYEAH` projects and we'll merge this :)

File: README.md (hunk 1 above)
Comment (techievena) [reply README.md]: @userzimmermann Don't you have that `@coala.io`type mail id?

Comment (userzimmermann) [reply README.md]: @Techievena `zimmermann@coala.io` :)

**Review by userzimmermann — APPROVED** [review]

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by adtac** [comment]
@rultor merge

**Comment by adtac** [comment]
@Techievena wanna rebase?

**Comment by techievena** [comment]
@adtac was off for a while. Rebasing now. :smile:

File: README.md (hunk 1 above)
Comment (userzimmermann) [inline README.md]: Oops! Forgot to check that :) Please really remove those 2 links above

**Review by userzimmermann — CHANGES_REQUESTED** [review]
@Techievena Thanks a lot for doing work of your team leader! ;) And for rebasing! But please remove those 2 outdated project links. Then I will merge it :)

**Review by userzimmermann — APPROVED** [review]

**Comment by userzimmermann** [comment]
@Techievena Wonderful! 🎉

**Comment by userzimmermann** [comment]
@rultor merge

**Pull request merged**